City-of-Helsinki/drupal-helfi-platform-config
# UHF-2723 translation helper

A drush command which creates the feature specific translation files. Translation files are created under each feature in translations folder

How to test:
- Setup a local environment
- Run this branch of the module: composer require drupal/helfi_platform_config:dev-UHF-2723_translation_helper
- Run drush command with feature name as a parameter: drush:helfi:create-translations {feature name here}

The translations which can be found in the feature should have changed.
